content/blog/2024/10/solus-4-6-released/index.md

```diff
@@ -0,0 +1,171 @@
+
+- XReader has replaced Evince
+- XViewer has replacecd Eye of GNOME
+- Engrampa has replaced File Roller
+
+Upstream release notes can be found [here](https://blog.buddiesofbudgie.org).
+
+## GNOME
+
+{{< altimg "GNOME.jpg" "/release-images/2024-10-14/" >}}
+
+Solus 4.6 GNOME Edition ships with GNOME 46.5.
+
+### Added Features
+
+- Global Search
+- Enhanced files app
+- Upgraded online accounts, now with OneDrive support
+- Remote login with RDP
+
+### Bug Fixes, cleanup & quality-of-life improvements
+
+- GNOME's file app
+- Settings improvements
+- Accessibility improvements
+
+Upstream release notes can be found [here](https://release.gnome.org/46/)
+
+## Plasma
+
+{{< altimg "Plasma.jpg" "/release-images/2024-10-14/" >}}
+
+Solus 4.6 Plasma Edition ships with the latest Plasma Desktop, KDE Frameworks, KDE Gear, and the KDE branch for QT. The most notable change is the introduction of Plasma 6. If you're looking for Wayland, Plasma still has you covered.
+
+Upstream release notes:
+
+- [Plasma 6.1.5](https://kde.org/announcements/plasma/6/6.1.5/)
+- [KDE Gear 24.08.1](https://kde.org/announcements/gear/24.08.1/)
+- [KDE Frameworks 6.6.0](https://kde.org/announcements/frameworks/6/6.6.0/)
+- [QT 6.7.3](https://code.qt.io/cgit/qt/qtreleasenotes.git/about/qt/6.7.3/release-note.md)
+- [sddm 0.21.0](https://github.com/sddm/sddm/blob/v0.21.0/ChangeLog)
+
+## XFCE
+
+{{< altimg "XFCE-Beta.jpg" "/release-images/2024-10-14/" >}}
+
+We continue to call our XFCE Desktop a *beta* experience, though we have been shipping this desktop since the release of Solus 4.5.
+
+The XFCE edition includes:
+
+- XFCE version 4.18
+- Mousepad 0.6.2
+- Parole 4.18.1
+- Ristretto 0.13.2
+- Thunar 4.18.11
+- Whiskermenu 2.8.3
+
+Since Solus 4.5, we've added the following to our default XFCE experience:
+
+- Engrampa has replaced File Roller
+- XReader is now the default PDF reader
+- The keyboard layout switcher plugin is now installed by default
+
+While not installed by default, a couple of other XFCE-related packages have been added since our last release:
+
+- xfdashboard
+- xfce4-battery-plugin
+- xfce4-clipman-plugin
+
+## Known Issues
+
+- [Plasma ISO Requires creation of a kwallet key file before connecting to a network](https://github.com/getsolus/packages/issues/1429)
+- When running `eopkg check`, linux-current and linux-lts may show as broken. These are false positives, and can be disregarded.
+- Users with the proprietary nVidia drivers and the 6.10 kernel may experience a black screen or errors on a black screen after waking the system from sleep.
```

Comment:
```suggestion
- Users with the proprietary NVIDIA drivers and the 6.10 kernel may experience a black screen or errors on a black screen after waking the system from sleep.
```
It's written in uppercase these days
https://www.nvidia.com/content/dam/en-zz/Solutions/about-us/documents/NVIDIA-Trademark-and-Logo-Usage-Guidelines.pdf